src/staticanalysis/bot/static_analysis_bot/config.py

```diff
@@ -65,8 +65,10 @@ def setup(self,
               cov_config=None):
         self.app_channel = app_channel
         self.download({
-            'cpp_extensions': frozenset(['.c', '.h', '.cpp', '.cc', '.cxx', '.hh', '.hpp', '.hxx', '.m', '.mm']),
```

Comment:
we don't need the header files in this context since using 'cpp_tenssions' we decide if we use clang and coverity based analysis. But in the case we only have a header in the revision we shouldn't start this kind of analysis since we don't have any compilation units to match against.